sql/core/src/test/resources/sql-tests/inputs/misc-functions.sql

```diff
@@ -8,3 +8,6 @@ select typeof(cast(1.0 as float)), typeof(1.0D), typeof(1.2);
 select typeof(date '1986-05-23'),  typeof(timestamp '1986-05-23'), typeof(interval '23 days');
 select typeof(x'ABCD'), typeof('SPARK');
 select typeof(array(1, 2)), typeof(map(1, 2)), typeof(named_struct('a', 1, 'b', 'spark'));
+
+-- get current_catalog
+select current_catalog();
```

Comment:
do we have a sql testing file for `current_database`?

Reply:
checked and not found

Reply:
how about we add a `current_database_catalog.sql` and test both of them there?

Reply:
I am okay with it.